### PR TITLE
Stride to step conversion

### PR DESCRIPTION
**Pull request by olwmc**
In light of #181 this PR reverses the change in syntax and semantics for the stride list comprehension. We are now back to the standard step notation with negative step values.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Documentation on Asteroid can be found at
 * A Python API allowing the developer to call the Asteroid interpreter from within a Python program and also embed
   Python code in an Asteroid program.
 
-* Eliminated the `stride` notation in list comprehensions and replaced it with the `step` notation.
+* Eliminated the `step` notation in list comprehensions and replaced it with the `step` notation.
```
Comment (lutzhamel) [inline README.md:16]: I think the first 'step' in this sentence needs to be 'stride' ...

Comment (olwmc) [reply README.md:16]: That's hilarious. I'll fix that.

**Comment by lutzhamel** [comment]
It's good to go. Nice work!

**Comment by olwmc** [comment]
@lutzhamel The documentation generation post push seems to run fine!

**Pull request merged**